### PR TITLE
Update libvpx from Chromium 114.0.5735.358

### PR DESCRIPTION
**Pull request by dahlstrom-g**
Issue: 326748668

### CODE REVIEW

File: third_party/libvpx/BUILD.gn [hunk 1]
```diff
@@ -127,10 +131,8 @@ if (current_cpu == "x86" || (current_cpu == "x64" && !is_msan)) {
   # symbols.
 
   source_set("libvpx_intrinsics_mmx") {
-    if (!use_cobalt_customizations) {
-      configs -= [ "//build/config/compiler:chromium_code" ]
-      configs += [ "//build/config/compiler:no_chromium_code" ]
-    }
+    configs -= [ "//build/config/compiler:chromium_code" ]
```
Comment (osagie98) [inline third_party/libvpx/BUILD.gn:134]: Do we still need to guard this with `use_cobalt_customizations`?

Comment (dahlstrom-g) [reply third_party/libvpx/BUILD.gn:134]: No.

File: third_party/libvpx/BUILD.gn [hunk 2]
```diff
@@ -108,7 +112,7 @@ if (current_cpu == "x86" || (current_cpu == "x64" && !is_msan)) {
     }
 
     if (!use_cobalt_customizations) {
-      defines = [ "CHROMIUM" ]
+    defines = [ "CHROMIUM" ]
```
Comment (dahlstrom-g) [inline third_party/libvpx/BUILD.gn:115]: Do you know what this does and whether excluding it is (still) necessary?

Comment (osagie98) [reply third_party/libvpx/BUILD.gn:115]: Discussed offline

**Review by osagie98 — APPROVED** [review]

**Pull request merged**